truescrub/db.py

```diff
@@ -95,7 +99,8 @@ def get_all_players():
     ORDER BY skill_mean - 2 * skill_stdev DESC
```

Comment:
Does SQLite not let you say "ORDER BY mmr DESC" after naming it earlier in the statement?

Reply:
It does.